Changelog.md

```diff
@@ -1,10 +1,16 @@
 # Motoko compiler changelog
 
+## FUTURE (2022-03-XX)
+
+* motoko (`moc`)
+
+  * Adds new flag `--suppress-metadata` to omit certain metadata sections from `actor` (and `actor class`) Wasm (#3164)
```

Comment:
hmm, maybe --omit-metadata is actually a better name for this?

Reply:
Yeah, why not?

Reply:
Could you add a line to the compiler options adoc file too?

Reply:
Done in 864326878c76c3ad09510f6370edbe5818a1ba1b and ed9cbcf03.

Reply:
And 47e57d527. PTAL.